examples/fitness/lib/settings.dart

```diff
@@ -4,6 +4,69 @@
 
 part of fitness;
 
+class _SettingsDialog extends StatefulComponent {
+  _SettingsDialogState createState() => new _SettingsDialogState();
+}
+
+class _SettingsDialogState extends State<_SettingsDialog> {
+  static final GlobalKey weightGoalKey = new GlobalKey();
+
+  InputValue _goalWeight;
+
+  void initState() {
+    super.initState();
+    _goalWeight = InputValue.empty;
```

Comment:
You can initialize this inline on line 14 because the initialization doesn't depend on `this`.